languages/core-standards-de_DE_formal.po

```diff
@@ -85,19 +85,51 @@ msgstr ""
 msgid "Show logout link"
 msgstr ""
 
-#: templates/cookie-notice.php:8
+#: templates/cookie-consent.php:8
+msgid "Butter cookies only"
+msgstr "Nur erforderliche Cookies"
+
+#: templates/cookie-consent.php:9
+msgid "Required functioanlity"
+msgstr "Erforderliche Funktionen"
+
+#: templates/cookie-consent.php:17
+msgid "Chocolate cookies"
+msgstr "Cookies für Statistiken"
+
+#: templates/cookie-consent.php:18
+msgid "Statistics"
+msgstr "Statistiken"
+
+#: templates/cookie-consent.php:22
+msgid "Space cookies"
+msgstr "Cookies zur Personalisierung"
```

Comment:
Major: The English words in the source code needs to contain the formal / neutral / regular words. The "funny" version must only be contained in the informal German translation.

We will most likely remove the funny version at some point, therefore the translations should not be based on it but the regular language instead.

Reply:
I guess this is done here... https://github.com/netzstrategen/wordpress-core-standards/pull/41